### ISSUE TITLE
Tag and release info missing for 0.4.*

### ISSUE THREAD

**Issue by mike-engel** [lucasconstantino/graphql-resolvers#21]
There's no git tags for any `0.4.x` releases, and no changelog info. I have no idea what's changed in these breaking versions.

**Comment by karltaylor** [comment]
Yes I agree it would be great if there could be a proper github release for `0.4.x`.

Here is a compare @mike-engel for `0.3.4...0.4.2`

https://github.com/lucasconstantino/graphql-resolvers/compare/c7b063b16c4c220263695fbcee081f3f37f65b7e...5c962a97825bd843821c14cb640a93c399c0d188
